views/album.ejs

```diff
@@ -9,7 +10,8 @@
 
   <% } %>
 <% } %>
-<h1><%= name %></h1>
+</div>
+<h1 class="title"><%= name %></h1>
```

Comment:
Not sure title is a suitably descriptive classname, as it could apply to any H1 on a site. Could we go with "albumTitle" maybe?